src/Endpoints/class-base-api-proxy.php

```diff
@@ -148,11 +147,7 @@ protected function get_data( WP_REST_Request $request, bool $require_api_secret
 		}
 
 		// A proxy with caching behavior is used here.
-		$response = $this->proxy->get_items( $params ); // @phpstan-ignore-line.
-
-		if ( false === $response ) {
-			return new stdClass();
-		}
```

Comment:
This was added while incorporating PHPStan but now after giving correct type we don't need this.